projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC
+
+#include "ltc_adc.h"
+#include "status.h"
+
+typedef void (*CurrentSenseCallback)(int32_t current, void *context);
+
+typedef struct {
+  int32_t voltage;  // Voltage in microvolts
+  int32_t current;  // Voltage in milliamps
+} CurrentSenseValue;
+
+// User-defined points for two-point calibration
+typedef struct {
+  CurrentSenseValue zero_point;
+  CurrentSenseValue max_point;
+} CurrentSenseCalibrationData;
+
+typedef struct {
+  LtcAdcStorage *adc_storage;
+  CurrentSenseCalibrationData *data;
+  int32_t current_value;
+  volatile uint8_t samples;
```

Comment:
This seems unnecessary. It should be fairly stable and the only point of zeroing is to give us a reference point. If we're off by the ADC's ripple that's not a huge deal.

Reply:
Removed